lang/java/avro/src/main/java/org/apache/avro/util/Utf8.java

```diff
@@ -119,16 +120,21 @@ public Utf8 setByteLength(int newLength) {
     }
     this.length = newLength;
     this.string = null;
-    this.hasHash = false;
+    this.hash = 0;
```

Comment:
Hello!  For consistency with [how Schema caches the hash](https://github.com/apache/avro/blob/42d81d70e7b9409d3b17fbcc9ea102876b45b945/lang/java/avro/src/main/java/org/apache/avro/Schema.java#L111), what do you think about using Integer.MIN_VALUE instead of 0?  Not a big deal, except that all "zeroed" byte arrays will otherwise have hashCode 0 and be recalculated every time.

Reply:
I personally prefer 0 because all fields get initialized as 0 in Java...
Would it be okay to leave unchanged?